<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(6, 101, 116);
color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLCDNumber" name="hourValue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>460</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="secondValue">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>460</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="minuteValue">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>460</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QPushButton" name="updateWeatherLocation">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Update</string>
    </property>
   </widget>
   <widget class="QLabel" name="imageLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="zipCodeEdit">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>30</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>98105</string>
    </property>
   </widget>
   <widget class="QLabel" name="weatherLabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>451</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>20</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="zipcode_label">
    <property name="geometry">
     <rect>
      <x>668</x>
      <y>34</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Zip Code</string>
    </property>
   </widget>
   <widget class="QLabel" name="slideshow_images">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>450</y>
      <width>291</width>
      <height>191</height>
     </rect>
    </property>
    <property name="text">
     <string>SLIDESHOW</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldHourValue">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>520</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldMinuteValue">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLCDNumber" name="worldSecondValue">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>520</y>
      <width>50</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="digitCount">
     <number>2</number>
    </property>
    <property name="value" stdset="0">
     <double>22.000000000000000</double>
    </property>
    <property name="intValue" stdset="0">
     <number>22</number>
    </property>
   </widget>
   <widget class="QLabel" name="weatherImages">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>130</y>
      <width>401</width>
      <height>281</height>
     </rect>
    </property>
    <property name="text">
     <string>weather</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>59</x>
      <y>530</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New York (EST)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>470</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Current Location (PST)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>420</y>
      <width>161</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Lato</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string>TOP 5 ANIMES</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>871</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
